packages/tocco-theme/src/ToccoTheme/defaultTheme.js

```diff
@@ -3,6 +3,9 @@ export default {
     paper: '#fff',
     primary: '#9E2124',
     text: '#4f5e60',
+    background: '#EDF1F5',
```

Comment:
i would be **very** restrictive when it comes to extending the theme.
Every var should serve a sole purpose. Maybe it's best to start with a documentation where you write for what each var is used for?
A theme should allow to style the application for every customer in their corporate colors.

Concern one: What is the difference between paper and background? What is used when is not clear to me.

Concern two: As said above a  theme should allow to set styling aspects of the application without assuming any color palette. So its would be possible to set primaryBlue to Pink? If there is no way around these two new vars i would at least give them a more generic name (and document well for what they are used for)